amazon/paapi.py

```diff
@@ -260,9 +266,10 @@ def search_products(self, item_count=10, item_page=1, items_per_page=10, keyword
             for x in range(3):
                 try:
                     # Wait before doing the request
-                    wait_time = 1 / self.throttling - (time.time() - self.last_query_time)
-                    if wait_time > 0:
-                        time.sleep(wait_time)
+                    if self.throttling:
```

Comment:
@sergioteula should pull the duplicated throttling logic into a helper function. It looks like the entire `for x in range(3)` is all duplicative.

Reply:
Also already added, I will update the pull request. Thanks for the suggestion!